app/components/forms/wizard/basic-details-step.js

```diff
@@ -269,6 +276,44 @@ export default Component.extend(FormMixin, EventWizardMixin, {
             }
           ]
         },
+        ticketMinPrice: {
+          identifier : 'min_price',
+          rules      : [
+            {
+              type   : 'empty',
+              prompt : this.l10n.t('Minimum price for donation tickets required')
+            },
+            {
+              type   : 'number',
+              prompt : this.l10n.t('Invalid number')
+            },
+            {
+              type   : 'integer[1..]',
+              prompt : this.l10n.t('Ticket price needs to be greater than zero')
+            },
```

Comment:
```js
{
              type   : 'number',
              prompt : this.l10n.t('Invalid number')
            },
            {
              type   : 'integer[1..]',
              prompt : this.l10n.t('Ticket price needs to be greater than zero')
            },
```
No need for two validations here too , Integer can be only used

Reply:
Removed the number validator.